meson_options.txt

```diff
@@ -0,0 +1,97 @@
+option(
+  'docs',
+  type: 'boolean',
+  value: true,
+  description: 'Whether to generate the API reference for libproxy'
+)
+
+option(
+  'tests',
+  type: 'boolean',
+  value: true,
+  description: 'Whether to compile test cases for libproxy'
+)
+
+option(
+  'dbus',
+  type: 'feature',
+  value: 'auto',
+  description: 'Whether to build D-Bus service'
+)
+
+option(
+  'dbus-system-user',
+  type: 'string',
+  value: 'root',
+  description: 'Which user should be used on D-Bus system session'
+)
```

Comment:
This is not OK. We should certainly not be running libproxy service as root.

Reply:
So leave default empty to set it to `libproxy` ?

Reply:
If that works, sure.

Reply:
Changed default to libproxy.